### PR TITLE
uchv-80 - added reference number step

### PR DESCRIPTION
**Pull request by gxxm**
* added step to route config
* added field and validation to fields config
* added checkbox group and dependent fields
* added field translations
* added acceptance tests

### CODE REVIEW

File: apps/complaints/acceptance/features/which-reference.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Which reference numbers step');
```
Comment (JoeChapman) [inline apps/complaints/acceptance/features/which-reference.js]: is it a page or a step?

File: apps/complaints/fields.js [hunk 2]
```diff
@@ -172,5 +172,43 @@ module.exports = {
...
+      toggle: 'ihs-reference',
+      child: 'input-text'
+    }]
+  },
+  'gwf-reference': {
+    dependent: {
+      field: 'reference-numbers',
+      value: 'gwf'
+    },
+    validate: ['required']
```
Comment (JoeChapman) [inline apps/complaints/fields.js]: do single validation rules need to be in array notation?

File: apps/complaints/acceptance/features/which-reference.js (hunk 1 above)
Comment (joefitter) [inline apps/complaints/acceptance/features/which-reference.js]: a step kind of is a page in our apps I guess?

File: apps/complaints/fields.js (hunk 2 above)
Comment (joefitter) [inline apps/complaints/fields.js]: it can be a string

**Comment by joefitter** [comment]
LGTM apart from the issue(s) that Joe raised

File: apps/complaints/acceptance/features/which-reference.js (hunk 1 above)
Comment (gxxm) [inline apps/complaints/acceptance/features/which-reference.js]: Everything previously has been step including Jira tickets

**Pull request merged**